src/all/kanjiku/src/eu/kanade/tachiyomi/extension/all/kanjiku/Kanjiku.kt

```diff
@@ -0,0 +1,102 @@
+package eu.kanade.tachiyomi.extension.all.kanjiku
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+
+class Kanjiku(
+    override val lang: String,
+    subDomain: String,
+) : ParsedHttpSource() {
+
+    override val name = "Kanjiku"
+
+    override val baseUrl = "https://${subDomain}kanjiku.net"
+
+    override val supportsLatest = false
+
+    override fun popularMangaRequest(page: Int): Request =
+        GET(baseUrl.toHttpUrl().newBuilder().apply { addPathSegment("mangas") }.build(), headers)
+
+    override fun popularMangaSelector(): String = ".manga_box"
+
+    override fun popularMangaFromElement(element: Element): SManga = SManga.create().apply {
+        setUrlWithoutDomain(element.absUrl("href"))
+        title = element.selectFirst(".manga_title")?.text() ?: "Title Error"
+        thumbnail_url = element.selectFirst("img")?.absUrl("src")
+    }
+
+    override fun mangaDetailsParse(document: Document): SManga = SManga.create().apply {
+        title = document.selectFirst(".manga_page_title")?.text() ?: "Title Error"
```

Comment:
```suggestion
        title = document.selectFirst(".manga_page_title")!!
```

Title should not have a fallback.